R/pstr_data.R

```diff
@@ -21,7 +21,7 @@
 #' * TODO: Where does it come from and what wrangling did you do?
 #'
 #' @author Linda Delacombaz.
-#' @source TODO
+#' @source Data from IEA's WEO 2022 scenario on GLO level.
```

Comment:
Thanks Linda! 
Can you please spell out the acronyms? I have no idea that all that means, ahaha. Or maybe point to the website of each of those organizations?

Reply:
ahahah yes, good idea !